app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -144,6 +168,45 @@ class ImageRegionSelectionInteractionViewTest {
   fun testImageRegionSelectionInteractionView_clickRegion3_clickRegion2_region2Clicked() {
     launch(ImageRegionSelectionTestActivity::class.java).use {
       it.onActivity {
+        it.window.decorView.layoutDirection = ViewCompat.LAYOUT_DIRECTION_LTR
```

Comment:
Rather than updating the ignored test, can we have a new test that specifically tests the issue we are trying to solve in this PR. This is we are not modifying the older tests which are already been ignored. 
Or can we confirm that with the new changes in the ignored test are passing the tests or not?

Reply:
For me on local the test `android://org.oppia.android.app.testing.ImageRegionSelectionInteractionViewTest.testImageRegionSelectionInteractionView_rtl_clickRegion3_clickRegion2_region2Clicked()` and `android://org.oppia.android.app.testing.ImageRegionSelectionInteractionViewTest.testImageRegionSelectionInteractionView_rtl_clickRegion3_region3Clicked()` are passing else all are ignored.

Reply:
> @veena14cs
> `Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing`

I have added tests screenshots for RTL specific. I haven't tested on Roboelectric I am unable to test on Arctic I need to check how to run tests on Roboelectric. Can you please check other ignored tests that pass for roboelectric or not as there is todo comment added for all the testcases that points to the issue[ #1611](https://github.com/oppia/oppia-android/issues/1611).